src/test/java/rx/internal/operators/OperatorSortedTest.java

```diff
@@ -0,0 +1,103 @@
+package rx.internal.operators;
+
+import org.junit.Test;
+import rx.Observable;
+import rx.Observer;
+import rx.functions.Func2;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+public class OperatorSortedTest {
```

Comment:
As there is no corresponding `OperatorSorted` I'd move these methods to `OperatorToSortedList`.